Gemfile

```
@@ -1,6 +1,6 @@
 source "https://rubygems.org"
 
-ruby '3.2.2'
+ruby(File.read(File.expand_path(".ruby-version", __dir__)))
```

we dryin'

...this is safe right?

---

TIL: we might be able to use this new syntax that Bundler [now supports](https://andycroll.com/ruby/read-ruby-version-in-your-gemfile/):

```suggestion
ruby file: ".ruby-version"
```

or if you feel strongly about parens...

```suggestion
ruby(file: ".ruby-version")
```

---

TIL! thanks. im all about parens but only when theres a chance of ambiguity